All_years/04-Social_Care/04-SDS_data.R

```diff
@@ -0,0 +1,167 @@
+#####################################################
+# Social Care SDS Data
+# Author: Jennifer Thom
+# Date: July 2022
+# Written on RStudio Server
+# Version of R - 3.6.1
+# Input - Data from Social care database DVPROD
+# Description - Process SDS data
+#####################################################
+
+# Load packages
+
+library(readr)
+library(dplyr)
+library(dbplyr)
+library(phsmethods)
+library(tidyverse)
+library(lubridate)
+
+
+# Set up------------------------------------------------------------------
+
+latest_update <- "Jun_2022"
+
+social_care_dir <- fs::path("/conf/hscdiip/SLF_Extracts/Social_care")
+
+
+# Read Demographic file----------------------------------------------------
+
+sc_demographics <- haven::read_sav(fs::path(
+  social_care_dir,
+  paste0("sc_demographics_lookup_", latest_update),
+  ext = "zsav"
+)) %>%
+  arrange(sending_location, social_care_id)
+
+
+# Query to database -------------------------------------------------------
+
+# set-up conection to platform
+db_connection <- phs_db_connection(dsn = "DVPROD")
+
+# read in data - social care 2 demographic
+sds_full_data <- tbl(db_connection, in_schema("social_care_2", "sds_snapshot")) %>%
```

Comment:
Doing this avoids the need to load dbplyr above.
```suggestion
sds_full_data <- tbl(db_connection, dbplyr::in_schema("social_care_2", "sds_snapshot")) %>%
```